src/components/AnimatedDropdown/index.tsx

```diff
@@ -5,7 +5,11 @@ import useResizeObserver from 'use-resize-observer'
  * @param open conditional to show content or hide
  * @returns Wrapper to smoothly hide and expand content
  */
-export default function AnimatedDropdown({ open, children }: React.PropsWithChildren<{ open: boolean }>) {
+export default function AnimatedDropdown({
+  open,
+  overflow = 'hidden',
```

Comment:
Why not make all animated dropdowns use `overflow: "visible"`? couldn't this just crop up in another menu in another language?